doc/MetadataAndQuery.md

```diff
@@ -0,0 +1,312 @@
+# Documentation for Query and Metadata interfaces
```

Comment:
I think of this doc as a temporary solution, when we define interface we can move those curl definitions to the postman